build-system/tasks/runtime-test.js

```diff
@@ -442,5 +473,6 @@ gulp.task('test', 'Runs tests', preTestTasks, function() {
     'config': '  Sets the runtime\'s AMP config to one of "prod" or "canary"',
     'coverage': '  Run tests in code coverage mode',
     'headless': '  Run tests in a headless Chrome window',
+    'local-changes': '  Run unit tests from files commited to the local branch',
```

Comment:
Curious what you think of inverting the logic here.

By default only run unit tests from files changed, use a flag to run against all.

Reply:
I think the default `gulp test` behavior should be to run all unit / integration tests (based on whether you use `--unit` or `--integration`), since that's the only way to make sure that changes to the core runtime didn't break anything. Note that `--local-changes` only works when you change a test file, and not when you change runtime code, since there's no way to reliably map each runtime files to a set of test files that it affects.

Reply:
Thanks. Makes sense!